### PR TITLE
 added grundy_number.cpp

### PR DESCRIPTION
**Pull request by Shalvi-Singhal**
#692 fixed.
@tarun26091999 @OjusWiZard  please review it.

### CODE REVIEW

File: Code/C++/grundy_number.cpp [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+
+    Time complexity: O(n)
+
+    Space complexity: O(n)
+
+*/
+
+
+
```
Comment (tarun26091999) [inline Code/C++/grundy_number.cpp:88]: remove these lines

**Review by tarun26091999 — CHANGES_REQUESTED** [review]
resolve conflicts.

**Comment by Shalvi-Singhal** [comment]
I think , I made some mistake while committing the change, @tarun26091999  can you please help me with resolving the conflict.
I have absolutely no idea on how to fix this.

**Comment by Shalvi-Singhal** [comment]
conflict resolved.

**Comment by Shalvi-Singhal** [comment]
commit https://github.com/Algo-Phantoms/Algo-Tree/pull/1546/commits/84dc7d79ebbbeb5ebfd2bd6a3242da67e99ee975 was done by mistake. how can I delete it?

**Comment by krishna-NIT** [comment]
> commit [84dc7d7](https://github.com/Algo-Phantoms/Algo-Tree/commit/84dc7d79ebbbeb5ebfd2bd6a3242da67e99ee975) was done by mistake. how can I delete it?

you git bash or github desktop  and then Resolve Issue of Readme, Arrange Question of Readme in alphabetical order

**Pull request closed without merging**